cmd/dra-example-kubeletplugin/state.go

```diff
@@ -130,6 +132,8 @@ func (s *DeviceState) Unprepare(claimUID string) error {
 		if err != nil {
 			return fmt.Errorf("unprepare failed: %v", err)
 		}
+	default:
+		return fmt.Errorf("unprepare failed: wrong prepared device type: %v", s.prepared[claimUID].Type())
```

Comment:
Can we just change this to:
```
		return fmt.Errorf("unknown device type: %v", devices.Type())
```
That is the only error from within this function. The unprepare is not called if the type is wrong.

Reply:
Done.